.gitignore

```diff
@@ -19,3 +19,4 @@ DerivedData
 Pods/
 Podfile.lock
 Carthage/
+GoogleMapsUtils.framework/
```

Comment:
Revert

Reply:
Unrelated to this (since I’m not sure how to add comments), but I think adding my Swift file to the program only allows my Swift HeatMapInterpolationPoints class to be used within the GoogleMapsUtils.xcodeproj rather than allowing it to be used in another Xcode project when someone imports GoogleMapsUtils.

Reply:
Ok I think I got it to work!! Let me know if it works on your end!